t/14-lb_req_dc_rr.t

```diff
@@ -183,9 +183,9 @@ local_dc must be a string
     }
 --- request
 GET /t
---- error_code: 500
---- error_log
-peer 127.0.0.3 data_center field must be a string
+--- error_code: 200
+--- no_error_log
+[error]
```

Comment:
This test should then be updated with the new behavior for missing `data_center` fields.